packages/common/src/store/pages/trending/reducer.ts

```diff
@@ -22,7 +22,9 @@ import {
   makeInitialState
 } from './lineup/reducer'
 
-const urlParams = new URLSearchParams(window.location.search)
+const urlParams = new URLSearchParams(
```

Comment:
ah interesting, this is cool to see where we make assumptions. since this is in common... i wonder if we should update this at som point?

Reply:
Yeah most definitely, shouldn't be relying on window.location in the reducer

Reply:
The pattern I've seen most often is to just throw any global access into its own module as a re-export such that you can do
`import {location} from './location';`

Then it's easy to automock it for testing.